README.md

```diff
@@ -123,3 +125,10 @@ The server currently offers 2 tools:
 docker build -t datalayer/jupyter-mcp-server .
 ```
 
+## Installing via Smithery
+
+To install Jupyter MCP Server for Claude Desktop automatically via [Smithery](https://smithery.ai/server/@datalayer/jupyter-mcp-server):
+
+```bash
+npx -y @smithery/cli install @datalayer/jupyter-mcp-server --client claude
```

Comment:
jupyter-mcp-server is not a npm module (it is a python package).

the above command will fail

Reply:
Does your MCP support being remote?

Once deployed on Smithery, clients will be able to connect to your MCP without any dependencies. Your entire package will be Dockerized.

If your MCP won't work remotely this may not work. We're working on local MCPs in the future, where clients will clone the Dockerized image hosted in the registry.

Reply:
> Does your MCP support being remote?

jupyter-mcp-server is designed to run remote (works also local).

My point was more that `@datalayer/jupyter-mcp-server` is a npm javascript package, which we don't plan to implement.

Reply:
Ah, to clarify, that is *not* an npm package, it's a Smithery package.

Here's how this command works

`npx -y @smithery/cli install [smithery package name]`

This asks Smithery CLI to install a particular package

`@datalayer/jupyter-mcp-server` is your smithery package ID that we've indexed.

URL: http://smithery.ai/server/@datalayer/jupyter-mcp-server